libbeat/autodiscover/providers/kubernetes/kubernetes_test.go

```diff
@@ -228,7 +228,8 @@ func TestEmitEvent(t *testing.T) {
 					"kubernetes": common.MapStr{
 						"namespace": "default",
 						"container": common.MapStr{
-							"name": "filebeat",
+							"name":  "filebeat",
+							"image": "elastic/filebeat:6.3.0",
```

Comment:
Is this field in `fields.yml`? I guess it may be there already because of the kubernetes module

Reply:
yes it is there https://github.com/elastic/beats/blob/2e4d292ee518fa9a2d3e3e9e0543bf498a165a52/libbeat/processors/add_kubernetes_metadata/_meta/fields.yml#L65, also visible in your screenshot

Reply:
👍